solution/server/tests/integration/mapbox-access.integ.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, it, expect } from '@jest/globals'
+import rp from 'request-promise'
+import { Address } from '../../src/types/types'
+
+const baseAddress = 'http://localhost:3000/graphql'
+
+describe('Mapbox access - queryAddress tests', () => {
```

Comment:
What are your thoughts on `Integration Tests` vs. `Unit Tests`?

Reply:
I tried to summarize my answer because I think we can talk a lot about this.

Unit tests focus is on a specif portion of the code and mocks all external calls, it tests if the isolated component logic works as it should with the inputs created for each test.

Integration tests are related to the communication between external and internal services, closer to a production environment. They make sure that the contracts are being respected by the code tested and checks if the services keep working as expected. Integration tests must work with unit tests.

I think both kinds of tests are important to ensure the correct behavior of the system.